docs/api_autopilot.md

```diff
@@ -0,0 +1,56 @@
+# API autopilot
+
+Kusk gateway supports API auto deployment. 
+
+Any REST API will typically consist of a Kubernetes pod(s) that is running the server and a Kubernetes service pointed to it.
+
+## It just works
+To expose a pre-existing REST API in the cluster, you would just have to execute this:
+
+```sh
+kubectl apply -f svc.yaml
+```
+Or edit an existing Kubernetes service to add the annotation `kusk-gateway/openapi-url` with a URL to the location of your OpenAPI
+
+## Under the hood
+
+Let's explain what is going on.
+
+We added a convenience method that will allow users to easily expose their REST API through Kusk gateway by using the `kusk-gateway/openapi-url` annotation, and here is how.
```

Comment:
I'd start with "Kusk-gateway has a convenient method..." 
as the docs shouldn't be aware of release timeframe